src/test/resources/LintWithWorkspaceTest/lint_a_basic_correct_message_with_default_config_and_the_kotlin_android_plugin_v2/buf.yaml

```diff
@@ -0,0 +1,3 @@
+version: v2
+modules:
+  - path: workspace
```

Comment:
For my own understanding - is `buf.work.yaml` just not a thing anymore in v2?

Reply:
From `https://buf.build/docs/configuration/v1/buf-work-yaml`:

`Buf now has a v2 configuration available, where this file is no longer used. `